src/api/leagueTeams-api.ts

```diff
@@ -8,9 +8,9 @@ const get = async (
   leagueId: number,
   season: string
 ): Promise<StandingsResponse> => {
-  if (isDevelop) {
-    return premierLeagueData as unknown as StandingsResponse;
-  }
+  // if (isDevelop) {
```

Comment:
Do baseApi wynieś to sobie jako export const isDevelopApi = import.meta.env.DEV && import.meta.env.VITE_API_IS_DEV_API === "true";